src/libs/contentful/adapters/ContentfulAdapterFactory.ts

```diff
@@ -0,0 +1,33 @@
+import { ContentTypes } from '../types'
+
+import { adaptAppV2VenuesModule } from './modules/adaptAppV2VenuesModule'
+import { adaptBusinessModule } from './modules/adaptBusinessModule'
+import { adaptCategoryListModule } from './modules/adaptCategoryListModule'
+import { adaptExclusivityModule } from './modules/adaptExclusivityModule'
+import { adaptHighlightOfferModule } from './modules/adaptHighlightOfferModule'
+import { adaptOffersModule } from './modules/adaptOffersModule'
+import { adaptRecommendationModule } from './modules/adaptRecommendationModule'
+import { adaptThematicHighlightModule } from './modules/adaptThematicHighlightModule'
+import { adaptTrendsModule } from './modules/adaptTrendsModule'
+import { adaptVenueMapModule } from './modules/adaptVenueMapModule'
+import { adaptVenuesModule } from './modules/adaptVenuesModule'
+import { adaptVideoCarouselModule } from './modules/adaptVideoCarouselModule'
+import { adaptVideoModule } from './modules/adaptVideoModule'
+
+export type ContentfulAdapter<Contentful, Module> = (module: Contentful) => Module | null
+
+export const contentfulAdapters: Record<string, ContentfulAdapter<any, any>> = {
```

Comment:
est-ce qu'on pourrait etre plus précis que string ?

```suggestion
export const contentfulAdapters: Record<ContentTypes, ContentfulAdapter<any, any>> = {
```